### PR TITLE
changes to allow restoration from remote urls

### PR DESCRIPTION
**Pull request by afaulconbridge**
This allows users to use this docker-compose and follow the procedure we describe for restoring elasticsearch at https://docs.targetvalidation.org/faq/spin-your-own-instance

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,14 +2,18 @@ version: "3.6"
...
     environment:
       # assign more memory to JVM 
-      - ES_JAVA_OPTS=-Xms12g -Xmx12g
-      - xpack.security.enabled=false
-      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms12g -Xmx12g"
+      - "xpack.security.enabled=false"
+      - "bootstrap.memory_lock=true"
+      #allow downloading from open targets google buckets
+      - repositories.url.allowed_urls=https://storage.googleapis.com/open-targets-data-releases/*
```
Comment (mkarmona) [inline docker-compose.yml]: I think elasticsearch is missing a list instead an element there. This is what is in the elasticsearch configuration that it is infrastructure elasticsearch folder repositories.url.allowed_urls: ["https://s3-eu-west-1.amazonaws.com/*","https://storage.googleapis.com/*"]

Comment (afaulconbridge) [reply docker-compose.yml]: But we don’t have anything on AWS anymore? I’ll take a look at the rest to see if thweres anything else from there we can add.

**Comment by mkarmona** [comment]
we don't have anything at aws finally but it will make others life easier
allowing this by default

File: docker-compose.yml (hunk 1 above)
Comment (afaulconbridge) [reply docker-compose.yml]: Fair enough, makes sense for others that might want to do their own backups / restores. I've added AWS URLs now.

**Pull request merged**